instrumentation/spring/spring-boot-autoconfigure/src/test/java/io/opentelemetry/instrumentation/spring/autoconfigure/resources/SpringConfigPropertiesTest.java

```diff
@@ -32,6 +32,7 @@ void shouldInitializeAttributesByMapInArow() {
     this.contextRunner
         .withConfiguration(AutoConfigurations.of(OpenTelemetryAutoConfiguration.class))
         .withPropertyValues(
+            "otel.metrics.exporter=none", // to suppress confusing error log
```

Comment:
Perhaps, metrics and logs exports should also be disabled for this test:

```
      "otel.traces.exporter=none",
      "otel.logs.exporter=none"
```

The default export interval is 1 minute for metrics, 5 s for traces and 1 s for logs.

Reply:
done :smile: